sdk/reservations/test-resources.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please delete this file, this is used for the live test in pipeline, and it isn't need in mgmt. plane currently.